docs/users/cli-usage.md

```diff
@@ -211,26 +211,25 @@ Quickstart instructions for database imports can be found under [Database Import
 
 #### TYPO3 Composer Setup Example
 
-```
+```bash
 mkdir my-typo3-site
 cd my-typo3-site
 ddev config --project-type=typo3 --docroot=public --create-docroot=true
 ddev start
-ddev composer create "typo3/cms-base-distribution:^9" --prefer-dist
+ddev composer create "typo3/cms-base-distribution:^10" --prefer-dist
+ddev exec touch public/FIRST_INSTALL
```

Comment:
Great addition. I wish we could count on Windows users having "touch", and this could just be `touch public/FIRST_INSTALL`

Reply:
Other option would be `echo "">public/FIRST_INSTALL` which also should work on every platform. But for me the touch approach looks more common.